core/variant_op.cpp

```diff
@@ -334,6 +334,13 @@ bool Variant::booleanize() const {
 		DEFAULT_OP_ARRAY_OP_BODY(m_prefix, m_op_name, m_name, m_type, !=, !=, true, false, false) \
 	}
 
+#define DEFAULT_OP_ARRAY_NEQ(m_prefix, m_op_name, m_name, m_type)                                \
+	CASE_TYPE(m_prefix, m_op_name, m_name) {                                                     \
+		if (p_b.type == NIL)                                                                     \
+			_RETURN(false)                                                                       \
```

Comment:
That doesn't seem consistent with the Array != logic at line 526.

Reply:
Right you are, fixed.